src/EnergyPlus/PlantComponent.hh

```diff
@@ -90,6 +90,11 @@ public:
     {
     }
 
+    virtual Real64 getDynamicMaxCapacity([[maybe_unused]] EnergyPlusData &state)
+    {
+        return 0.0;
+    }
```

Comment:
Virtual function returns 0 if equipment model does not have an override function.